tests/core/test_sync_monitor_stateful.py

```diff
@@ -187,7 +187,8 @@ async def _bob_update_file(self, wid, file_path, create_file=False):
                 await wfs.touch(file_path)
             else:
                 data = self.get_next_data()
-                await wfs.write_bytes(file_path, data)
+                if await wfs.exists(file_path):
```

Comment:
In which case would we be here with `file_path` not existing ?

I have the feeling we this `if exists` is not a good idea given we rely on the tested component to determine if we should do the test...

(besides this implies `wfs.write_bytes` behavior has changed... is this expected ?)

Reply:
> this implies wfs.write_bytes behavior has changed... is this expected ?

Yes, as explained in this comment: https://github.com/Scille/parsec-cloud/pull/1383#discussion_r474081558

I simply adapted the code to match the previous behavior, should I investigate this in more depth?

Reply:
The `create_file` param seems to indicate the file should already be present, so I think we should remove the `wfs.exists check` here